reason_4.0/lib/core/entity_delegates/abstract.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Abstract base entity delegate
+ *
+ * @package reason
+ * @subpackage entity_delegates
+ */
+/**
+ * Abstract base entity delegate
+ *
+ * All entity delegates should inherit from this class
+ */
+abstract class EntityDelegate
```

Comment:
This doesn't match case used in subclass definitions. We should either use "entityDelegate" or change them to "EntityDelegate".